shared/app/index.native.tsx

```diff
@@ -88,21 +88,22 @@ const Keybase = () => {
   ensureStore()
   if (!_store) return null // never happens
 
+  // TODO if we use it, add it here
+  // <React.StrictMode>
```

Comment:
i'm doing a lot of profiling and not being in strict mode is better, i think its fine to leave this off for now